qcfractal/interface/client.py

```diff
@@ -132,6 +138,18 @@ def _request(self,
 
         return r
 
+    @property
+    def mock_network_error(self):
+        """
+        Boolean for debugging purposes. When set to True, requests to the server always raise a network error without
+        actually making an attempt to connect
+        """
+        return self._mock_network_error
+
+    @mock_network_error.setter
+    def mock_network_error(self, value):
```

Comment:
Do we need accessors here? Might just to set directly via `client._mock_network_error = True`. A little tacky, but would rather not increase the number of user facing functions for testing.